test/addons/argocd/start

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+import json
+
+from drenv import kubectl
+from drenv import minikube
```

Comment:
This should not be imported in addons, we don't want dependencies on minikube.

Reply:
done